pkg/collector/corechecks/snmp/internal/report/report_device_metadata_test.go

```diff
@@ -406,7 +406,7 @@ func Test_batchPayloads(t *testing.T) {
 	for i := 0; i < 350; i++ {
 		interfaces = append(interfaces, metadata.InterfaceMetadata{DeviceID: deviceID, Index: int32(i)})
 	}
-	payloads := batchPayloads("my-ns", "127.0.0.0/30", collectTime, 100, device, interfaces)
+	payloads := batchPayloads("my-ns", "127.0.0.0/30", collectTime, 100, device, interfaces, nil)
```

Comment:
Do we have a test where we pass `topologyLinks`?

Reply:
Good catch, tested with some content in topologyLinks here https://github.com/DataDog/datadog-agent/pull/13887/commits/235846ac1c92f5fbc20d7942326bd9a58d32da12

This test is mainly meant to the batching logic.